Tools/jit/template.c

```diff
@@ -69,9 +69,11 @@ do {                                                \
     } while (0)
 
 #undef LLTRACE_RESUME_FRAME
-#define LLTRACE_RESUME_FRAME() \
-    do {                       \
-    } while (0)
+#ifdef Py_DEBUG
+#define LLTRACE_RESUME_FRAME() frame->lltrace = 0;
```

Comment:
```suggestion
#define LLTRACE_RESUME_FRAME() (frame->lltrace = 0)
```

This will be followed by a semi colon when used.